vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java

```diff
@@ -79,16 +81,27 @@ public void typeText(WebElement element, String text)
      */
     public void clearText(WebElement element)
     {
-        performWithHideKeyboard(element::clear);
+        performWithHideKeyboard(element, WebElement::clear);
     }
 
-    private void performWithHideKeyboard(Runnable runnable)
+    private void performWithHideKeyboard(WebElement webElement, Consumer<WebElement> performOnElement)
     {
-        runnable.run();
+        performOnElement.accept(webElement);
 
         // https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Commands/FBCustomCommands.m#L107
         if (genericWebDriverManager.isIOSNativeApp() && realDevice)
         {
+            String tagName = webElement.getTagName();
+            /**
+             * Tap on 'Return' doesn't close the keyboard for XCUIElementTypeTextView element, the only way to close the
```

Comment:
shouldn't it be stated somewhere in the docs also?

Reply:
Also, I propose to describe the strategy to the work with such fields in the documentation